.github/actions/ginkgo/main-prs.yaml

```diff
@@ -1,7 +1,7 @@
 # This file contain which k8s-versions should be executed for each PR
+# When upgrading k8s version, increment each minor version of each line by one.
 ---
 k8s-version:
   - "1.31"
   - "1.30"
-  - "1.29"
   - "1.28"
```

Comment:
just curious why we are removing 1.29, and not 1.28 💭

Reply:
@michi-covalent 1.28 is the minimal k8s version that we support, so that's why we are keeping it. The PR that bumped k8s version from 1.30 to 1.31 (https://github.com/cilium/cilium/pull/34463) didn't perform this step. Now, the reason why we are skipping the 1.29 is because the coverage is the same as 1.30.